src/components/Accordion/Accordion.tsx

```diff
@@ -30,6 +36,12 @@ export const Accordion = ({
           iconVariant,
         }}
       >
+        <AccordionHeader
+          actions={actions}
+          subtitle={subtitle}
+        >
+          {headerTitle}
+        </AccordionHeader>
```

Comment:
This change, together with the change to the interface is a breaking change, and it makes the component less flexible as we're restricting composability.

Previously we could do something like this:
```tsx
<Accordion>
  {isDeleted ? (
    <AccordionHeader actions={<RestoreButton />}>Deleteditem</AccordionHeader>
  ) : (
    <AccordionHeader actions={<DeleteButton />}>Active item</AccordionHeader>
  )}
  <div>some children</div>
</Accordion>
``` 

Instead we now have to duplicate the entire `Accordion`:
```tsx
{isDeleted ? (
  <Accordion title="DeletedItem" actions={<RestoreButton />}>
  	<div>some children</div>
  </Accordion>
) : (
  <Accordion title="ActiveItem" actions={<DeleteButton />}>
  	<div>some children</div>
  </Accordion>
)}
``` 

Not sure why this was changed? I do think that this change should be reverted.

Reply:
Ok, I was told that this was something you'd discussed changing and that I was to supposed to do it here, but in that case I'll change it back 😅

Reply:
Ok 🙂 I can't remember having had such a discussion, but I also tend to forget things, so not impossible 😅 It does however seem odd, as we should try to make our components composable over "prop based" where it makes sense. There are usecases for the prop-based approach though and passing an Icon to inputfield f.ex may be one of those, since we there will need full control over how the icon should be displayed. It might be possible to solve via composition as well though. Let's try and clarify on Slack